system-addon/test/unit/unit-entry.js

```diff
@@ -0,0 +1,11 @@
+const req = require.context(".", true, /\.test\.js$/);
+const files = req.keys();
+
+// This exposes sinon assertions to chai.assert
+sinon.assert.expose(assert, {prefix: ""});
```

Comment:
This is so we can do `assert.calledOnce`, etc., the same we do in our old mocha tests